examples/quickstart/main.rs

```diff
@@ -117,19 +122,19 @@ async fn main() {
     let mut pairs: HashMap<String, ProtocolComponent> = HashMap::new();
     let mut amounts_out: HashMap<String, BigUint> = HashMap::new();
 
-    let mut protocol_stream = ProtocolStreamBuilder::new(&tycho_url, Chain::Ethereum)
+    let mut protocol_stream = ProtocolStreamBuilder::new(&tycho_url, chain)
         .exchange::<UniswapV2State>("uniswap_v2", tvl_filter.clone(), None)
-        .exchange::<UniswapV3State>("uniswap_v3", tvl_filter.clone(), None)
-        .exchange::<EVMPoolState<PreCachedDB>>(
-            "vm:balancer_v2",
-            tvl_filter.clone(),
-            Some(balancer_pool_filter),
-        )
-        .exchange::<UniswapV4State>(
-            "uniswap_v4",
-            tvl_filter.clone(),
-            Some(uniswap_v4_pool_with_hook_filter),
-        )
+        // .exchange::<UniswapV3State>("uniswap_v3", tvl_filter.clone(), None)
+        // .exchange::<EVMPoolState<PreCachedDB>>(
+        //     "vm:balancer_v2",
+        //     tvl_filter.clone(),
+        //     Some(balancer_pool_filter),
+        // )
+        // .exchange::<UniswapV4State>(
+        //     "uniswap_v4",
+        //     tvl_filter.clone(),
+        //     Some(uniswap_v4_pool_with_hook_filter),
+        // )
```

Comment:
Need to test with this uncommented now that I've deployed all executors

Reply:
I would in fact uncomment them out one-by-one and test execution on each of them for sanity purposes.

Reply:
yes that's what I did for mainnet as well!